cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -331,10 +334,24 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
-  @Test def use_relationship_index() {
+  @Test def use_text_index() {
```

Comment:
do we want to update to/add text index tests to the ref card as well?

Reply:
Good point, thanks!